Upstream-Status: Pending

Fix these errors:
| automake: warnings are treated as errors
| data/ui/Makefile.am:18: warning: '%'-style pattern rules are a GNU make extension
| src/Makefile.am:10: warning: compiling 'pcmanfm.c' with per-target flags requires 'AM_PROG_CC_C_O' in 'configure.ac'
| autoreconf: automake failed with exit status: 1
| ERROR: autoreconf execution failed.
Signed-Off-By: Nitin A Kamble <nitin.a.kamble@Intel.com>
2012/07/09

Index: pcmanfm-0.9.10/configure.ac
===================================================================
--- pcmanfm-0.9.10.orig/configure.ac
+++ pcmanfm-0.9.10/configure.ac
@@ -13,6 +13,7 @@ AC_PROG_CC
 AM_PROG_LIBTOOL
 AM_PROG_CC_STDC
 AC_HEADER_STDC
+AM_PROG_CC_C_O
 
 # Checks for header files.
 AC_HEADER_STDC
Index: pcmanfm-0.9.10/data/ui/Makefile.am
===================================================================
--- pcmanfm-0.9.10.orig/data/ui/Makefile.am
+++ pcmanfm-0.9.10/data/ui/Makefile.am
@@ -15,6 +15,6 @@ EXTRA_DIST = \
 	$(NULL)
 
 # Purge GtkBuilder UI files
-%.ui: %.glade
+.glade.ui:
 	cp $< $@
 #	$(top_builddir)/src/xml-purge $@
